packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/index.md

```diff
@@ -9,18 +9,26 @@ title: Basic sign-in flow using the password factor
 
 <StackSelector class="cleaner-selector"/>
 
-The basic user sign-in request is the simplest of all use cases and is the first use case that you should try after you install the SDK.
+This guide covers a basic user sign-in request, which is the simplest of all use cases and is the first use case that you should try after you install the SDK. The flow diagram and steps describe how to build a simple sign-in form and how to authenticate the credentials.
 
-This use case requires the password factor.
-
-<div class="common-image-format">
+---
+**Learning outcomes**
+Understand how to implement basic sign-in using Okta Identity Engine.
 
-![Password factor only indicator](/img/oie-embedded-sdk/factor-password-only.png)
+**What you need**
+* [Okta org](https://developer.okta.com/signup/oie.html)
+* [Sample app](/docs/guides/oie-embedded-common-download-setup-app), downloaded and set up
 
-</div>
+**Sample code**
+n/a
+---
```

Comment:
In the rendered version, this horizontal rule is not rendering, and on a few others as well. again, probably fixed by line breaks.

Reply:
Yes, it is fixed with the line breaks.